SteamKit2/SteamKit2/Steam/Handlers/SteamAuthTicket/TicketInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.IO.Hashing;
+
+namespace SteamKit2
+{
+    /// <summary>
+    /// Represents a valid authorized session ticket.
+    /// </summary>
+    public sealed partial class TicketInfo : IDisposable
```

Comment:
That's not what I meant (the partial here is pointless), add an extra `public sealed partial class SteamAuthTicket`, so that it becomes SteamKit2.SteamAuthTicket.TicketInfo`

Reply:
Ah, sorry, haven't even noticed that